src/msg/async/rdma/Infiniband.cc

```diff
@@ -780,10 +773,16 @@ Infiniband::MemoryManager::MemoryManager(CephContext *c, Device *d, ProtectionDo
     // allocate initial rxbuf_pool size as 2 * receive_queue_len if
     // it's possible which could avoid expanding pool upon receiving
     // the first packet.
-    rxbuf_pool(&rxbuf_pool_ctx, sizeof(Chunk) + c->_conf->ms_async_rdma_buffer_size,
-              (c->_conf->ms_async_rdma_receive_buffers >= 2 * c->_conf->ms_async_rdma_receive_queue_len ||
-               c->_conf->ms_async_rdma_receive_buffers <= 0) ?
-               2 * c->_conf->ms_async_rdma_receive_queue_len : c->_conf->ms_async_rdma_receive_buffers)
+    rxbuf_pool(&rxbuf_pool_ctx,
+               // nrequest_size: chunk size
+               sizeof(Chunk) + cct->_conf->ms_async_rdma_buffer_size,
```

Comment:
Could you please not to mix cosmetic change with a fix? It's very hard to follow.

And here, please, make a separate static function for all the calculations with normal ifs, because here ternary operator is definitely overused. Impossible to read.